src/providers/Application/Context.tsx

```diff
@@ -18,9 +18,9 @@ export const ApplicationContext = createContext<ApplicationData>({
   applicationSections: [],
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   getApplicationByProjectId: (projectId: number) => undefined,
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-empty-function
   setSelectedApplication: (applicationId: number) => {},
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-empty-function
   reload: (onReload?: () => void) => {},
```

Comment:
Another option for these is something like this:
```ts
{
  // no-op
  return;
}
```
But I don't think that it necessarily provides benefit over just disabling it for one line

Reply:
Other alternative is disabling and enabling similar to `ParticipantProjectSpeciesContext.tsx`